### ISSUE TITLE
invalid export json?

### ISSUE THREAD

**Issue by wizardnet972** [palantir/tslint#1636]
### Bug Report

- __TSLint version__: `3.15.1`
- __TypeScript version__: `2.0.3`
- __Running TSLint via__: Node.js API

#### TypeScript code being linted

```ts
cmd> tslint src/**/*.ts -t json -o results.json
```

with `tslint.json` configuration:

#### Actual behavior
```
[
    {
        "endPosition": {
            "character": 2,
            "line": 7,
            "position": 162
        },
        "failure": "file should end with a newline",
        "name": "src/client/app/app.routes.ts",
        "ruleName": "eofline",
        "startPosition": {
            "character": 2,
            "line": 7,
            "position": 162
        }
    },
    {
        "endPosition": {
            "character": 13,
            "line": 2,
            "position": 58
        },
        "failure": "expected an assignment or function call",
        "name": "src/client/app/app.routes.ts",
        "ruleName": "no-unused-expression",
        "startPosition": {
            "character": 7,
            "line": 2,
            "position": 52
        }
    },
    {
        "endPosition": {
            "character": 13,
            "line": 2,
            "position": 58
        },
        "failure": "Missing semicolon",
        "fix": {
            "innerRuleName": "semicolon",
            "innerReplacements": [
                {
                    "innerStart": 58,
                    "innerLength": 0,
                    "innerText": ";"
                }
            ]
        },
        "name": "src/client/app/app.routes.ts",
        "ruleName": "semicolon",
        "startPosition": {
            "character": 13,
            "line": 2,
            "position": 58
        }
    }
][
    {
        "endPosition": {
            "character": 19,
            "line": 4,
            "position": 132
        },
        "failure": "The selector of the component \"BoardComponent\" should be named kebab-case (https://goo.gl/mBg67Z)",
        "name": "src/client/app/tic-tac-toe/components/board.component.ts",
        "ruleName": "component-selector-name",
        "startPosition": {
            "character": 12,
            "line": 4,
            "position": 125
        }
    }
]
```

#### Expected behavior

missing comma, invalid json (JSON.parse).

```
   "startPosition": {
            "character": 13,
            "line": 2,
            "position": 58
        }
    }
] <!---here missing comma , ---!>[
    {
        "endPosition": {
            "character": 19,
            "line": 4,
            "position": 132
        },
```

**Comment by adidahiya** [comment]
I believe this a duplicate of #379, which is fixed on the latest master & `v4.0.0-dev.0` via #1472
